model/src/main/java/org/projectnessie/model/CommitMeta.java

```diff
@@ -170,4 +203,16 @@ public Instant deserialize(JsonParser p, DeserializationContext ctxt) throws IOE
       return Instant.parse(p.getText());
     }
   }
+
+  private static final class CoauthorsList {
```

Comment:
Nit: rename

Reply:
Suggestion? :)

Reply:
I may steer is the direction of deriving `getAuthor()` from the list, so this may disappear :)